neuralforecast/common/_base_auto.py

```diff
@@ -112,10 +113,23 @@ def __init__(
     ):
         super(BaseAuto, self).__init__()
         self.save_hyperparameters()  # Allows instantiation from a checkpoint from class
-        config["h"] = h
+
+        assert (
```

Comment:
I think that it is convenient to change these asserts for if conditions with Exception messages.
The message helps user's debugging.
```
        if not self.fitted:
            raise Exception("This model instance is not fitted yet, Call fit method.")
```